src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+.html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
+.page-body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You don't need to add classes to `html` and `body` tags.
You can style them using tags.

Also, default padding of the body is already 0.